vip_spec.xsd

```diff
@@ -126,12 +124,32 @@ reference elements for polling location
               <xs:element name="WhereDoIVoteUrl" type="xs:anyURI" minOccurs="0" />
               <xs:element name="WhatIsOnMyBallotUrl" type="xs:anyURI" minOccurs="0" />
               <xs:element name="RulesUrl" type="xs:anyURI" minOccurs="0" />
-              <xs:element name="VoterServices" type="InternationalizedText" minOccurs="0" />
-              <!-- Note: The "Hours" element is being deprecated and will be removed
-                   in future versions of VIP.
-                -->
-              <xs:element name="Hours" type="InternationalizedText" minOccurs="0" />
```

Comment:
Actually, nevermind. I just remembered that the Contact data has HoursOpenId in it.